src/pages/profiles/[userId].tsx

```diff
@@ -62,12 +64,13 @@ export default function Page() {
   const [profile, setProfile] = useState<UserProfile>();
   useEffect(() => setProfile(old?.profile), [old]);
 
-  const updateProfile = (k: keyof UserProfile, v: string) => {
+  const updateProfile = (k: keyof UserProfile, v: string | ImageParams) => {
     const updated = {
       ...profile,
       [k]: v,
     };
     if (!v) delete updated[k];
+    if (k === "照片链接" && !v) delete updated["照片参数"];
```

Comment:
why do you need this line?

Reply:
without this line, remove link will not remove params. Next pic will still use the old params.

Reply:
ah I see!